### PR TITLE
I18n improvements

### PR DESCRIPTION
**Pull request by bosskovic**
All the changes are fully backward compatible, so anyone updating from previous version of the gem would see no difference at all, and yet it will be helpful for those people who need to localize the app.

### CODE REVIEW

**Comment by carlosramireziii** [comment]
@bosskovic Would you be willing to rebase this so it can get another review? 

cc @nickcharlton

**Comment by bosskovic** [comment]
@carlosramireziii rebased. Please let me know if I can do anything else to help with this. Btw, we have this in production for 3 apps since late Decemeber, and it all works fine. It would be great if I could get rid of the fork and switch back to the main repo.

cc @nickcharlton

**Comment by bosskovic** [comment]
I rebased and resolved conflicts again.

**Comment by nickcharlton** [comment]
Hi @bosskovic! I've only just been able to look at this now. Sorry to be a pain; could I get you to rebase again?

I notice we've got a few merges of `master` in the history here, could you drop those whilst you do it?

File: app/helpers/administrate/application_helper.rb [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
   module ApplicationHelper
-    PLURAL_MANY_COUNT = 2.1
-
     def render_field(field, locals = {})
       locals.merge!(field: field)
       render locals: locals, partial: field.to_partial_path
     end
 
-    def display_resource_name(resource_name)
+    def display_resource_name(resource_name, count = 2.1)
```
Comment (bosskovic) [inline app/helpers/administrate/application_helper.rb]: I see that a constant `PLURAL_MANY_COUNT` was introduced. I kept my solution, but I set the default for the count argument to 2.1.

**Comment by bosskovic** [comment]
hello @nickcharlton,
I rebased, you may take a look. Some tests are failing because of the difference in managing pluralization (the tests expect hard coded counts).
I don't know how to clean up git history quickly, can't you just squash all commits if you decide to merge?
Let me know if there's anything else.

**Comment by willian** [comment]
Hey guys, do we have any update o this PR?

**Comment by febeling** [comment]
This would be useful.

**Comment by nickcharlton** [comment]
I'm going to close this as it's quite a lot of changes and translations have changed since this was opened and so there's a lot of conflicts.

If you'd be able to open a collection of smaller PRs, we'll be able to get them merged sooner.

**Comment by pablobm** [comment]
I had a closer look. I think this is a good PR. Unfortunately it needs rebasing and some conflicts solved. Would someone be willing to look into that?

**Comment by nickcharlton** [comment]
I'm keen to get these improvements in, as that's the general opinion here. But I just timeboxed an hour to rebase this and it's particularly difficult with the amount of changes since this was originally opened and the amount of commits.

If anyone else would like to give this a go, I'd be all ears. If not, I'm going to close this next week (2020-04-03) and we can revisit it again.

**Comment by nickcharlton** [comment]
Closing as I said I would a week ago — I'd suggest manually replaying the work against current `master` if someone wanted to pick this up. It's still valuable, but quite difficult after a few years of big changes.

**Pull request closed without merging**